src/main/java/mediathek/filmlisten/FilmeImportierenDateiThread.java

```diff
@@ -12,7 +16,12 @@
 
     public FilmeImportierenDateiThread(String pfad, ListeFilme listeFilme, int days, IDownloadAction downloadAction,
                                        IAction onFinished) {
-        this.pfad = pfad;
+        // is directory-path of 'pfad' really a directory (relative/fix) ? (-> convert 'pfad' to directory if it is a file...)
+        File file_pfad_dir = new File(getDirnameFromPathname(pfad));
+        if(file_pfad_dir.isDirectory())
+            this.pfad = pfad;
+        else
+            this.pfad = System.getProperty("user.dir") + File.separator + file_pfad_dir;
```

Comment:
As I worte in discriptoin, I don't know the Backend process in sufficient details, so "user.dir" ist only a gues that I use in several programs and always works.
More or less its only an example to use a relative or absolute directory, as problem discribed. It can even identify directories and files platform independent.

Reply:
From where can I get the abo folder?